Source/WebInspectorUI/UserInterface/Controllers/CodeMirrorCompletionController.js

```diff
@@ -94,6 +94,8 @@ WI.CodeMirrorCompletionController = class CodeMirrorCompletionController extends
         }
 
         this._completions = completions;
+        if (completions[0] instanceof WI.QueryResult)
+            this._completions = completions.map(WI.CSSCompletions.getCompletionText);
```

Comment:
actually, why not just always do this since it already has nice behavior when everything is a string?
```js
this._completions = completions.map(WI.CSSCompletions.getCompletionText);
```

also, i think we should move `getCompletionText` to a more generic class (e.g. `WI.Completions`) since `WI.CodeMirrorCompletionController.prototype.updateCompletions` is technically not specific to any language (i.e. it's sorta a layering violation